commercetools-models/src/test/java/io/sphere/sdk/carts/CartLikeTest.java

```diff
@@ -0,0 +1,51 @@
+package io.sphere.sdk.carts;
+
+import io.sphere.sdk.json.SphereJsonUtils;
+import io.sphere.sdk.utils.MoneyImpl;
+import org.junit.Before;
+import org.junit.Test;
+
+import static io.sphere.sdk.models.DefaultCurrencyUnits.EUR;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CartLikeTest {
+
+    private Cart cartWithTaxes;
+    private Cart cartWithoutTaxes;
+
+    @Before
+    public void setUp() throws Exception {
+        cartWithTaxes = SphereJsonUtils.readObjectFromResource("cart-with-taxes.json", Cart.class);
+        cartWithoutTaxes = SphereJsonUtils.readObjectFromResource("cart-without-taxes.json", Cart.class);
+    }
+
```

Comment:
One option would be to leave in the JSON file only the minimum required info, but that has the risk of losing useful data for future tests or implementations of the methods (and generating again the data is time-costly).